code/controllers/subsystem/air.dm

```diff
@@ -268,7 +268,7 @@ var/datum/subsystem/air/SSair
 		active_turfs -= T
 		if (T.blocks_air)
 			continue
-		T.Initalize_Atmos(times_fired)
+		T.Initialize_Atmos(times_fired)
```

Comment:
It really makes me sad to see you include unrelated changes like this in your prs.

Today mostly because my testing/profiling setup requires I edit this file, and editing this file 4 or 5 times is lame and it would have been nice if i could have just copy and pasted my changes in as one block, but now i can't, and its not even for a good reason.

You fuck yourself over with conflicts, you fuck other people over with conflicts, and its really not necessary.